components/x-gift-article/package.json

```diff
@@ -28,7 +28,7 @@
     "@financial-times/o-loading": "^5.2.1",
     "@financial-times/o-message": "^5.2.1",
     "@financial-times/o-normalise": "^3.2.2",
-    "@financial-times/o-typography": "^7.2.2",
+    "@financial-times/o-typography": "^7.4.1",
```

Comment:
**Non-blocking comment**: I believe the `^` already brings you to the latest major version

Reply:
yeah it shouldn't make a deference but just for the sake of clarification since the new version contains the new professional branding